hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1453,6 +1456,23 @@ message PartInfo {
     required uint64 size = 4;
 }
 
+/**
+ Stores the raw payload of rpc request, used in OmRPCLoadGenerator
+ */
+message EchoRPCRequest {
+    optional bytes payloadReq = 1;
+    optional int32 payloadSizeResp = 2;
+    optional bool isEmptyResp = 3;
```

Comment:
Same as above `payloadSizeResp` == 0 should server as the boolean.

Reply:
sure!

Reply:
Hey Ritesh! I change the code a little bit. When option isEmptyResponse is [true](https://github.com/apache/ozone/pull/3614/files#diff-c2423ca69889970743c3fc1e72385459675b8ac9dd4d80227ea4b92090625690R99), I set the response payload to 0, and don't send the boolean [flag](https://github.com/apache/ozone/pull/3614/files#diff-c2423ca69889970743c3fc1e72385459675b8ac9dd4d80227ea4b92090625690R118) anymore. Feel free to let me know how you think!